Solutions/OpenVPN/Data Connectors/OpenVPN_Syslog.json

```diff
@@ -0,0 +1,141 @@
+{
+    "id": "OpenVPN",
+    "title": "OpenVPN Server",
+    "publisher": "OpenVPN",
+    "descriptionMarkdown": "The [OpenVPN](https://github.com/OpenVPN) data connector provides the capability to ingest OpenVPN Server logs into Microsoft Sentinel.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**OpenVpnEvent**](https://aka.ms/sentinel-openvpn-parser) which is deployed with the Microsoft Sentinel Solution.",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "OpenVPN",
+        "baseQuery": "OpenVpnEvent"
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Sources",
+        "query": "OpenVpnEvent\n | summarize count() by tostring(SrcHostname)\n | top 10 by count_"
```

Comment:
Hi @spsocprime, SrcHostname is not defined in the parser but still being used in data connector query. Because of that query is failing. Can you please add SrcHostname in parser as well.
Thanks.

Reply:
fixed